modules/olm-installing-from-operatorhub-using-cli.adoc

```diff
@@ -13,10 +13,17 @@ endif::[]
 
 :_mod-docs-content-type: PROCEDURE
 [id="olm-installing-operator-from-operatorhub-using-cli_{context}"]
-= Installing from OperatorHub using the CLI
+= Installing from OperatorHub by using the CLI
 
 Instead of using the {product-title} web console, you can install an Operator from OperatorHub by using the CLI. Use the `oc` command to create or update a `Subscription` object.
 
+For `SingleNamespace` install mode, you must also ensure an appropriate Operator group exists in the related namespace. An Operator group, defined by an `OperatorGroup` object, selects target namespaces in which to generate required RBAC access for all Operators in the same namespace as the Operator group.
+
+[TIP]
+====
+For most usage, the web console method of this procedure is preferred because it automates tasks in the background, such as handling the creation of `OperatorGroup` and `Subscription` objects automatically when choosing `SingleNamespace` mode.
```

Comment:
I'm not sure _usage_ is the right word. I'm prolly wrong, so feel free to ignore.

```suggestion
In most cases, the web console method of this procedure is preferred because it automates tasks in the background, such as handling the creation of `OperatorGroup` and `Subscription` objects automatically when choosing `SingleNamespace` mode.
```